README.md

```diff
@@ -24,37 +31,73 @@ strapi:
     # Collections, key is used to access in the strapi.collections
     # template variable
     collections:
-        # Example for a "articles" collection
-        articles:         
+        # Example for a "Photo" collection
+        photos:
             # Collection name (optional)
-            type: article
+            # type: photos
             # Permalink used to generate the output files (eg. /articles/:id).
-            permalink: /articles/:id/
+            permalink: /photos/:id/
             # Layout file for this collection
-            layout: strapi_article.html
+            layout: photo.html
             # Generate output files or not (default: false)
             output: true
 ```
 
+This works for the following collection *Photo* in Strapi:
+
+| Name    | Type  |
+| ------- | ----- |
+| Title   | Text  |
+| Image   | Media |
+| Comment | Text  |
+
+### Authentication
+
+To access non Public collections (and by default all Strapi collections are non Public) you must to generate a token inside your strapi instance and set it as enviromental variable `STRAPI_TOKEN`.
+
+It is recommended that you will use new Content API tokens for this task: https://strapi.io/blog/a-beginners-guide-to-authentication-and-authorization-in-strapi 
+
 ## Usage
 
-This plugin provides the `strapi` template variable. This template provides access to the collections defined in the configuration.
+This plugin provides the `strapi` template variable. This template provides access to the collections defined in the configuration.  
 
 ### Using Collections
 
 Collections are accessed by their name in `strapi.collections`. The `articles` collections is available at `strapi.collections.articles`.
 
-To list all documents of the collection:
+To list all documents of the collection ```_layouts/home.html```:
 
 ```
-{% for post in strapi.collections.articles %}
-<article>
-    <header>
-        {{ post.title }}
-    </header>
-    <div class="body">
-        {{ post.content }}
-    </div>
-</article>
-{% endfor %}
+---
```

Comment:
In Strapi 4+, we have a **Media Library**, is it possible to download images from the media gallery into the **uploads/:year/:month** ?